src/static/js/views/faq.html

```diff
@@ -15,17 +15,17 @@ <h4>Frequently Asked Questions</h4>
         What is Software Discovery Tool?
       </div>
       <div class="list-group-item" ng-show="questionState(1)">
-        <p>Software Discovery Tool, helps you to search for available packages across OS distributions in a single user interface. Current version of Software Discovery Tool is v1.5.28</p>
+        <p>Software Discovery Tool, helps you to search for available packages across OS distributions supported by the s390x architecture in a single user interface.</p>
       </div>
       <div class="list-group-item list-group-item-info ng-class:{'active': questionState(2), 'glyphicon-minus':questionState(2), 'glyphicon-plus':!questionState(2)};"  ng-click="clickQuestion(2)"> <!--ng-class="[list-group-item, { 'active':questionState(2) } ]"-->
         Which operating systems does Software Discovery Tool support?
       </div>
       <div class="list-group-item" ng-show="questionState(2)">
-        <p>Currently, Software Discovery Tool supports the following operating systems:
+        <p>Currently, Software Discovery Tool contains data for the following operating systems:
         <ul>
-            <li>SUSE Linux Enterprise Server (12 SP3,SP4,SP5,15, 15 SP1, SP2)</li>
-            <li>Ubuntu (19.04,19.04,20.04,20.10)</li>
-            <li>Red Hat Enterprise Linux 6.10/7.6/7.7/7.8/7.9/8.0/8.1/8.2/8.3 </li>
+            <li>IBM z/OS</li>
+            <li>Debain 10(buster)</li>
+            <li>ClefOS Epel 7 </li>
```

Comment:
We have the data for "base" rather than "epel" and it's still under review, so I'd delete this one for now.